joomla/joomla-cms
# Introtext Show/Hide parameter is ignored in Blogview / Fix

### Summary of Changes

This PR adds a simple if clause to check if the introtext is set to show or hide in the Parameters
### Testing Instructions

Before Applying the patch:
In your Joomla! Test environment navigate to the subpage /article-category-blog 
Setup the Menuitem in Backend to hide introtext there
Check the Frontend, The introtext still shows up.

After Applying the patch:
1) Setup the Menuitem in Backend to hide introtext in "article category blog".
The introtext should be hidden now.

2) Setup the Menuitem to show introtext in "article category blog".
The introtext should be shown.
### Documentation Changes Required
